README.md

```diff
@@ -8,46 +8,58 @@ Codebase for Sparkle, brought to you by Sparkle.
 
 Clone the repo and cd into it
 
-```
+```bash
 git clone https://github.com/sparkletown/sparkle
 cd sparkle
 ```
 
-Now, obtain the `.env` file for the environemnt (eg. staging) and save it (eg. `.env.staging`)
+<!-- Now, obtain the `.env` file for the environemnt (eg. staging) and save it (eg. `.env.staging`) -->
 
-```
+Now, obtain the `.env` file for the environment (ask someone on the engineering team) and save it.
+
+<!-- ```
 # copy in the .env files
 # symlink the staging .env file
 ln -s .env.staging.local .env
-```
+``` -->
 
-Now you're ready to start the server
+Install the npm modules
 
-```
+```bash
 npm i
+```
+
+Now you're ready to start the server! ✨
+
+```bash
 npm test
 npm start
 ```
 
 You won't need to in dev, but you can also test builds:
 
-```
+```bash
 npm run build
 ```
 
 ### Firebase functions
 
+To do this, you must first be added to the firebase environment.
+In a new terminal enter the following commands.
+
 ```bash
 npm install -g firebase-tools@latest
```

Comment:
Technically this doesn't need to be installed globally, as we include it as part of our dev dependencies, so it will be installed/usable in that way regardless.

```suggestion
# While not necessary (as we already include it in our devDependencies), you can install the firebase-tools globally if desired
# npm install -g firebase-tools@latest

```